### PR TITLE
Add support for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by apartridge**

### CODE REVIEW

File: continuous-integration/setup/setup_build_and_test.sh [hunk 1]
```diff
@@ -17,10 +17,15 @@ apt-yes install \
     clinfo \
     wget \
     python3-pip \
-    python-catkin-tools \
     unzip \
     || exit $?
 
+if [[ "$UBUNTU_VERSION" == "20.04" ]]; then
+    apt-yes install python3-catkin-tools python3-osrf-pycommon || exit $?
```
Comment (runenordmo) [inline continuous-integration/setup/setup_build_and_test.sh:24]: I guess `python3-osrf-pycommon` became a dependency in U20/Neotic?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -15,13 +15,20 @@ stages:
...
+        ros_distros: ['ros:noetic-ros-base-focal']
+        compilers: ['g++-9', 'g++-10', 'g++-11', 'clang++-9', 'clang++-10', 'clang++-11']
   - stage: Build_And_Test_Newest_Zivid_Melodic
     jobs:
     - template: continuous-integration/azure-pipelines/templates/build_and_test.yml
       parameters:
         zivid_versions: ["${{ variables['newest_zivid_version'] }}"]
         ros_distros: ['ros:melodic-ros-base-bionic']
-        compilers: ['g++-7', 'g++-8', 'g++-9', 'clang++-7']
+        compilers: ['g++-7', 'g++-8', 'g++-9', 'clang++-7', 'clang++-8']
```
Comment (runenordmo) [inline azure-pipelines.yml:31]: Can you comment reason to test/support these specific compilers for melodic/neotic?
Maybe include it in the commit message

**Comment by runenordmo** [comment]
No more comments from me :+1:

File: continuous-integration/setup/setup_build_and_test.sh (hunk 1 above)
Comment (apartridge) [reply continuous-integration/setup/setup_build_and_test.sh:24]: This is taken from https://answers.ros.org/question/353113/catkin-build-in-ubuntu-2004-noetic/ and is needed for some reason.

File: azure-pipelines.yml (hunk 2 above)
Comment (apartridge) [reply azure-pipelines.yml]: expanded commit message

**Review by runenordmo — APPROVED** [review]
Nice update of the commit message about which compilers should be tested for each distro :+1:
Changes LGTM now!

**Pull request merged**